app/templates/partials/summary/unit.html

```diff
@@ -0,0 +1 @@
+{{answer.unit|format_unit (answer.value)}}
```

Comment:
The syntax for this looks odd, if two arguments are used they should be passed like `format_unit(answer.unit, answer.value)`. This is done for the `format_start_end_date` filter (using a context processor).